src/store/types.d.ts

```diff
@@ -1,3 +1,7 @@
+export type SupportedMediaType = 'audio' | 'image'
+export type SupportedSearchType = 'all' | SupportedMediaType
```

Comment:
One day, when we have more time to do TypeScript fiddling, I'd love to derive this from the JS configuration object. Something like:

```
type MediaType = 'audio' | 'image' | 'video'
type SearchType = 'all' | MediaType

const supportedMediaTypes = ['audio' | 'image'] as const
const supportedSearchTypes = ['all', ...supportedMediaTypes] as const
type SupportedMediaType = typeof supportedMediaTypes[number]
type SupportedSearchType = typeof supportedSearchTypes[number]
```

Then we'd only have to change the actual JS array and the types would magically follow suit :star_struck: